scarpe-components/lib/scarpe/components/calzini/para.rb

```diff
@@ -76,6 +76,19 @@ def text_specific_styles(props)
       raise Shoes::Errors::InvalidAttributeValueError, "Unexpected underline type #{props["underline"].inspect}!"
     end
 
+    s1[:'font-style'] = case props["emphasis"]
+    when "normal"
+      "normal"
+    when "oblique"
+      "oblique"
+    when "italic"
+      "italic"
+    when nil
+      "normal" # Set a default value for nil
```

Comment:
Right now if props["emphasis"] isn't set, you're specifically setting it to "normal". That means an em tag which doesn't manually set that property will show up as not-italic. You could not set the property or set it to nil in cases where none is specified and I *think* that would fix the problem.

Reply:
Also, since you're setting this on s1 instead of s2 (which looks right to me) , probably good to put the code in the upper part of the method where we're setting s1 properties. That's a really minor thing, though.

Reply:
Alright. Thank you for review.

Reply:
Hey, I have made the suggested changes. Could you please review them?